tempest/api/workloadmgr/retention/test_retention.py

```diff
@@ -77,6 +79,19 @@ def test_retention(self):
                 reporting.add_test_step("Retention", tvaultconf.FAIL)
                 LOG.debug("Retention didn't work!!")
                 raise Exception("Retention failed")
+
+            snapshot_id1 = snapshot_ids[0]
+            self.mount_path = self.get_mountpoint_path()
+            self.snapshot_found = self.check_snapshot_exist_on_backend(self.mount_path, workload_id, snapshot_id1)
+            LOG.debug(f"snapshot_found: {self.snapshot_found}")
+            if not self.snapshot_found:
+                reporting.add_test_step("Retention verified on backend, snapshot is deleted",
+                                        tvaultconf.PASS)
+            else:
+                LOG.debug("Retention failed")
+                reporting.add_test_step("Retention falied on backend, snapshot is not deleted",
```

Comment:
Can we say backup target, instead of backend?

Reply:
updated the text